content/agent/kubernetes/daemonset_setup.md

```diff
@@ -238,29 +260,37 @@ Learn more about this in [the Docker log collection documentation][9].
 
 ### Trace Collection
 
-To enable [Trace collection][10] with your DaemonSet:
+To enable trace collection with your DaemonSet:
 
-1. Set the `DD_APM_ENABLED` variable to true in your *env* section:
+1. Set the Node IP and port as environment variables for your application containers:
 
     ```
-    (...)
-      env:
-        (...)
-        - name: DD_APM_ENABLED
-            value: "true"
-    (...)
+    apiVersion: apps/v1
+    kind: Deployment
+    ...
+        spec:
+          containers:
+          - name: <CONTAINER_NAME>
+            image: <CONTAINER_IMAGE>/<TAG>
+            env:
+              - name: DD_AGENT_HOST
+                valueFrom:
+                  fieldRef:
+                    fieldPath: status.hostIP
+              - name: DD_TRACE_AGENT_PORT
```

Comment:
We can remove this, it's `8126` by default and if they change this, it's advanced usage.